`DisplayEntity` overrides `getWidth` in `Entity`, which is final

This is also true for `getHeight`. Relevant mappings are:
https://github.com/QuiltMC/quilt-mappings/blob/2174f57f80ef2eb43451afe3681a40a173b7aa58/mappings/net/minecraft/entity/decoration/DisplayEntity.mapping#L56
https://github.com/QuiltMC/quilt-mappings/blob/2174f57f80ef2eb43451afe3681a40a173b7aa58/mappings/net/minecraft/entity/Entity.mapping#L204

This caused a crash in my development environment, obviously because final methods cannot be overridden:
```
Caused by: java.lang.IncompatibleClassChangeError: class net.minecraft.entity.decoration.DisplayEntity overrides final method net.minecraft.entity.Entity.getHeight()F
```
`getHeight` in `DisplayEntity` is `final` and `private`, though, so I might be missing something. In fabric yarn mappings, `getHeight` is instead named `getDisplayHeight` to avoid this issue.

--- a/mappings/net/minecraft/entity/decoration/DisplayEntity.mapping
+++ b/mappings/net/minecraft/entity/decoration/DisplayEntity.mapping
@@ -58,7 +58,7 @@ CLASS net/minecraft/unmapped/C_zdeutotk net/minecraft/entity/decoration/DisplayE
 	METHOD m_mfdnbnif getBillboardRenderConstraints ()Lnet/minecraft/unmapped/C_zdeutotk$C_cozbiatm;
 	METHOD m_mhzmmfsl getTransformation (Lnet/minecraft/unmapped/C_pyoaoolj;)Lnet/minecraft/unmapped/C_qvfggwhk;
 		ARG 0 tracker
-	METHOD m_mkgxqfrt getWidth ()F
+	METHOD m_mkgxqfrt getDisplayWidth ()F
 	METHOD m_nbotmszd setTransformation (Lnet/minecraft/unmapped/C_qvfggwhk;)V
 		ARG 1 transformation
 	METHOD m_ogmsufqg getInterpolationProgress (F)F
@@ -77,7 +77,7 @@ CLASS net/minecraft/unmapped/C_zdeutotk net/minecraft/entity/decoration/DisplayE
 		ARG 1 brightness
 	METHOD m_wmplpzcr interpolate (F)Lnet/minecraft/unmapped/C_qvfggwhk;
 		ARG 1 delta
-	METHOD m_xnqmzrtk getHeight ()F
+	METHOD m_xnqmzrtk getDisplayHeight ()F
 	METHOD m_xqfovtgz (Lcom/mojang/datafixers/util/Pair;)V
 		ARG 1 pair
 	METHOD m_xytfwvza getShadowRadius ()F

